out/build/x64-Debug/performous.rc

```diff
@@ -0,0 +1,33 @@
+#include <windows.h>

```

Comment:
There's still accidentally commited files left in.

Reply:
Removed